Documentation/raspi-libgpiodv2.md

```diff
@@ -0,0 +1,99 @@
+# Use libgpiodv2 on a Raspberry Pi
```

Comment:
Thanks a bunch for adding docs and instructions :). That said, I wonder if we should instead make this guide also consider previous versions of libgpiod. For instance, someone who is not familiar with the native library and just want to  use that driver, they may incorrectly think that they have to go through these installation steps in order to do so, which is not true as most distros already come with libgpiod, just not the newer version.

I'm not suggesting this becomes a big libgpiod guide where all of that is explained, (nor that it is your fault that we don't have good docs for this today :P), but since we are introducing some docs with the PR we should make sure these aren't confusing for users who just want to use the libgpiod driver.

Reply:
And, final doc comment, but it would be good if we find somewhere in our existing docs to link to this page as I don't think it is currently being linked in your changes.

Reply:
I think we can agree on the user not having to select a specific driver version, loading it automatically under the hood. I will change these docs to only give necessary information about libgpiod.